source/refined-github.css

```
@@ -22,8 +22,9 @@
 	padding: 0 !important;
 }
 
+/* Modify only the left and right padding (#3606) */
```

```suggestion
/* Modify only the left and right padding #3606 */
```

👍 Thanks for putting a comment!